src/manifold/src/face_op.cpp

```diff
@@ -316,4 +316,28 @@ CrossSection Manifold::Impl::Project() const {
 
   return CrossSection(polys).Simplify(precision_);
 }
+
+std::vector<int> Manifold::Impl::ReflexFaces(double tolerance) const {
+  std::unordered_set<int> uniqueReflexFaceSet;
```

Comment:
Why not just count concave edges like in the [other PR](https://github.com/elalish/manifold/pull/668/files#diff-f39cd492c2bc62c44078967bdc98150cee9c55934e499dc1e941ad9ee27ef27f)?

Reply:
It’s a holdover from the Voronoi Convex Decomposition which needed the face version 🫠

I can look at switching it soon…

Reply:
Alrighty; added an "IsConvex" check that wraps the Genus check and the check from the other PR.